### DISCUSSION TITLE
fuzzy skin improves overhang performance!

### DISCUSSION THREAD

**Discussion by doriantgt** [supermerill/SuperSlicer#4508] [General]
I used the fuzzy skin setting to print a 60deg overhang. The idea is that the filament shrinks back to the wall allowing for steeper overhangs. This was printed with abs in a heated build chamber with no cooling fan.

I got the idea from this video. https://www.youtube.com/watch?v=B0yo-o47688

It would be cool if there were a way to do a rectilinear fill as a perimeter for overhangs. I believe the weaving motion would improve the overhang performance. 

![IMG_0115](https://github.com/user-attachments/assets/237ef3ae-be2d-4b27-bab1-bf50f26625ab)
![IMG_0114](https://github.com/user-attachments/assets/584be1b5-0c53-4eea-b67d-e636a156715e)
![IMG_0113](https://github.com/user-attachments/assets/a65cde1e-2a9c-4fd1-ba34-200131b4e508)
![IMG_0112](https://github.com/user-attachments/assets/32c1083f-5d2b-48d3-9fda-8dde20201993)

**Comment by dzid26** [comment]
I can print 75deg easily without fuzzy skin, so what's the big deal

**Reply by supermerill** [reply]
that depends on the filament, nozlle, cooling, nozzle...
What you can do now, maybe you won't later.

**Reply by dzid26** [reply]
My point is he didn't demonstrate claimed improvement.

**Reply by supermerill** [reply]
True
@doriantgt 
If you can show us the same object, printed with the same filament on the same printer with the same settings, but the fuzzy skin, to be able to see the improvement?
Also, if you can print with a ~gray-ish filament (at least somethins not too dark), it's better to take picture and see lines.

**Reply by doriantgt** [reply]
I’ll run that later today

**Comment by doriantgt** [comment]
Another example of perimeter pattern to combat overhang. https://www.instagram.com/reel/DPtyVLFk-iW/?utm_source=ig_web_copy_link
